gradle/build-hadoop-distros.gradle

```diff
@@ -28,6 +28,7 @@ project('spring-xd-hadoop:hadoop24') {
 		runtime ("org.apache.hadoop:hadoop-mapreduce-client-jobclient:$hadoop24Version")
 		runtime ("org.apache.hadoop:hadoop-streaming:$hadoop24Version")
 		runtime ("org.apache.hadoop:hadoop-yarn-common:$hadoop24Version")
+		configurations.runtime.exclude(group: 'org.xerial.snappy')
```

Comment:
These will not get picked up by the maven plugin. If they need to be excluded from the pom need to specify the modules explicitly